actionpack/lib/action_controller/log_subscriber.rb

```diff
@@ -23,7 +23,8 @@ def process_action(event)
         additions = ActionController::Base.log_process_action(payload)
         status = payload[:status]
 
-        if status.nil? && (exception_class_name = payload[:exception].first)
+        if status.nil? && payload[:exception].present?
```

Comment:
How about to use `payload[:exception]&.first`?

```suggestion
        if status.nil? && (exception_class_name = payload[:exception]&.first)
```

Reply:
Thanks, I actually tried that first, and then I saw test failures and assumed it was because I used the `&.` operator (even though I know Rails requires Ruby 2.3+). The test failures don't seem related.